apps/backend-e2e/src/e2e/auth.cy.ts

```diff
@@ -15,140 +15,87 @@ describe('AUTHENTIFICATION TESTS', () => {
       - Check self-profile of editor user with wrong jwt.
       - Check self-profile with correct jwt.
    */
-
-  /*
-      SENSIBLE DATA MUST BE SENT WITH COMMAND LINE AND OVERRIDE FOLLOWING DEFAULT ONES :
-   */
-  let jwtUser = {};
-  let idUserAdmin;
-  let idUserEditor;
-
-  /*
-      TESTS
-   */
-  it('Auth -> Connecting with incorrect user.', () => {
+  it('Should fail when trying to connect without any credentials 1/2.', () => {
```

Comment:
having the same test name with `X/2` is not very descriptive, I think you should instead specify the differences between both tests. Here calling GET vs POST